app/services/check_cached_follows.rb

```diff
@@ -0,0 +1,33 @@
+class CheckCachedFollows
```

Comment:
I vote for namespacing this to `Follows::` - I think eventually our `app/services` folder should be grouped by section or functionality in someway, with less classes in the top level